lib/ui/widgets/password_text_field.dart

```diff
@@ -32,13 +34,13 @@ class _PasswordFormFieldState extends State<PasswordFormField> {
                     color: Colors.white, width: borderRadiusOnFocus)),
             focusedErrorBorder: OutlineInputBorder(
                 borderSide: BorderSide(
-                    color: Colors.amberAccent, width: borderRadiusOnFocus)),
+                    color: getErrorTextColor(), width: borderRadiusOnFocus)),
```

Comment:
```suggestion
                    color: errorTextColor, width: borderRadiusOnFocus)),
```

If you apply the change below 👇